ui-kit/src/main/res/drawable/ic_radio_button_checked.xml

```diff
@@ -0,0 +1,11 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:pathData="M2,12C2,6.48 6.48,2 12,2C17.52,2 22,6.48 22,12C22,17.52 17.52,22 12,22C6.48,22 2,17.52 2,12ZM7,12C7,9.239 9.239,7 12,7C14.761,7 17,9.239 17,12C17,14.761 14.761,17 12,17C9.239,17 7,14.761 7,12Z"
+        android:fillColor="#2B2B2B"
```

Comment:
use color from pallete.
I guess this one is @color/night

check other icons too please